usr/bin/playtronos-systemreport

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# System Report
+#
+# Generates a system report archive which can contain useful information for
+# reporting bugs and troubleshooting.
+
+if [ $EUID -ne 0 ]; then
+  echo "$(basename "$0") must be run as root"
+  exit 1
+fi
+
+ENDCOLOR='\e[0m'
+RED='\e[0;31m'
+PURPLE='\e[0;35m'
+CYAN='\e[0;36m'
+
+TARGET_USER=playtron
+REPORT_NAME="report-$(date +%Y-%m-%d-%s)"
+REPORT_DIR_BASE="$(mktemp -d)"
+REPORT_DIR="${REPORT_DIR_BASE}/${REPORT_NAME}"
+mkdir -p "${REPORT_DIR}"
+
+echo -e "${CYAN}Generating system report...${ENDCOLOR}"
+
+# Hardware
+echo -e "  Collecting ${RED}hardware${ENDCOLOR} report..."
+dmesg >"${REPORT_DIR}/dmesg.log"
+lspci -tv -nn >"${REPORT_DIR}/lspci.out"
+lsusb -t -v >"${REPORT_DIR}/lsusb.out"
+cat /proc/bus/input/devices >"${REPORT_DIR}/input-devices.out"
+hwctl system-info >"${REPORT_DIR}/system.json" 2>/dev/null
+
+# Firmware
+fwupdmgr get-devices >"${REPORT_DIR}/firmware-devices.out" 2>&1
+fwupdmgr get-updates -y >"${REPORT_DIR}/firmware-updates.out" 2>&1
+
+# Processes
+echo -e "  Collecting ${RED}process${ENDCOLOR} report..."
+ps aux >"${REPORT_DIR}/processes.out"
+
+# Input
+echo -e "  Collecting ${RED}input${ENDCOLOR} report..."
+timeout 0.3 evtest >"${REPORT_DIR}/evtest.out" 2>&1
+{
+  echo "InputPlumber State"
+  echo ""
+  echo "\$ inputplumber devices list"
+  inputplumber devices list
+  echo "\$ inputplumber device 0 info"
+  inputplumber device 0 info
+  echo "\$ inputplumber device 0 intercept get"
+  inputplumber device 0 intercept get
+  echo "\$ inputplumber device 0 targets list"
+  inputplumber device 0 targets list
+} >"${REPORT_DIR}/inputplumber.out" 2>&1
+
+# Power/TDP/Battery
+echo -e "  Collecting ${RED}power${ENDCOLOR} report..."
+busctl introspect org.shadowblip.PowerStation /org/shadowblip/Performance/GPU/card0 >"${REPORT_DIR}/powerstation.out"
+upower -d >"${REPORT_DIR}/battery.out" 2>&1
+
+# System logs
+echo -e "  Collecting ${RED}services${ENDCOLOR} report..."
+journalctl -b 0 >"${REPORT_DIR}/journal.log"
+journalctl -b 0 -u inputplumber >"${REPORT_DIR}/inputplumber.log"
+journalctl -b 0 -u powerstation >"${REPORT_DIR}/powerstation.log"
+sudo -u "${TARGET_USER}" journalctl --user -u playserve -b >"${REPORT_DIR}/playserve.log"
+sudo -u "${TARGET_USER}" journalctl --user -u gamescope-dbus -b >"${REPORT_DIR}/gamescope-dbus.log"
+sudo -u "${TARGET_USER}" journalctl --user -u "gamescope-session-plus@${TARGET_USER}" -b >"${REPORT_DIR}/gamescope-session.log"
+
+# Audio
+echo -e "  Collecting ${RED}audio${ENDCOLOR} report..."
+sudo -u "${TARGET_USER}" wpctl status >"${REPORT_DIR}/audio.out" 2>&1
```

Comment:
This wasn't working for me. It appears the command was still ran as root for some reason because the log file had this:

```
Could not connect to PipeWire
```